events/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from common.models import TimestampedModel, UUIDPrimaryKeyModel
+
+
+class Event(UUIDPrimaryKeyModel, TimestampedModel):
+    name = models.CharField(verbose_name=_("name"), max_length=64)
```

Comment:
As we don't know what kind of names they want to give to the events, probably better to be safe than sorry and add longer `max_length` to start with, how about 255? :)

Reply:
Agree, 64 seems a bit short here